### PR TITLE
fix: ambianic-performance-optimization

### PR DESCRIPTION
**Pull request by sniperadmin**
this is the initial report before starting optimization:
image

I will keep it as a reference for later :)

### CODE REVIEW

**Comment by sniperadmin** [comment]
i hate comment linting 😄 
it hunts `feat` && `refactor`

**Comment by sniperadmin** [comment]
Still working in progress, when it finishes, I will clear conflicts

**Comment by sniperadmin** [comment]
Congrats! Boosted performance by 10
image

File: src/App.vue [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <!-- App.vue -->
 <template>
   <v-app>
-    <AppFrame v-if="!$route.meta.isHome">
-      <router-view />
-    </AppFrame>
-    <router-view v-else />
+    <keep-alive>
```
Comment (sniperadmin) [inline src/App.vue:4]: this will cash the router for the performance

File: src/views/About.vue [hunk 2]
```diff
@@ -1,27 +1,20 @@
...
-            <v-list-item-content>
-              <v-list-item-title class="headline">
-                Cozy at Home
-              </v-list-item-title>
-              <v-list-item-subtitle>
-                via Ambient Intelligence
-              </v-list-item-subtitle>
-            </v-list-item-content>
-          </v-list-item>
+          <v-card-title>
```
Comment (sniperadmin) [inline src/views/About.vue]: reusable component are magic!

File: package.json [hunk 3]
```diff
@@ -35,6 +35,7 @@
     "moment": "^2.24.0",
     "node": "^13.12.0",
     "peerjs": "^1.1.0",
+    "prerender-spa-plugin": "^3.4.0",
```
Comment (sniperadmin) [inline package.json:38]: this is very important to prerender the components,.. becuase the pnp is consuming time to load the app!

**Comment by sniperadmin** [comment]
this is the tree node for current build
image

**Comment by sniperadmin** [comment]
ok so far this branch is complete with the ui

**Comment by ivelin** [comment]
Cool stuff! Have you noticed Netlify is not able to deploy this branch. Not clear if its an error on their end. I see build timeout issues in the Netlify logs (which are public access).

**Comment by sniperadmin** [comment]
let me check them

**Comment by sniperadmin** [comment]
might be from their end,.. i read that the max time for build time is 15 mins,
though, im suspecting the code-linter as it might be a reason

not sure yet

**Comment by sniperadmin** [comment]
I do think that the webpack-analyzer is causing the build timeout!
image

**Comment by sniperadmin** [comment]
I think i may start a fresh one

File: package.json (hunk 3 above)
Comment (ivelin) [reply package.json:38]: Nice. Worth a try. It probably won't apply to the timeline page, but others that are more static in nature can benefit.

**Comment by ivelin** [comment]
> I think i may start a fresh one

Yes, maybe easier if your local merge with master doesn't succeed.
Package-lock is such a common PITA to deal with.

**Comment by sniperadmin** [comment]
FYI this branch is continued on #365

**Pull request closed without merging**